specification/resourcemover/resource-manager/Microsoft.Migrate/preview/2019-10-01-preview/resourcemovercollection.json

```diff
@@ -1537,6 +1570,22 @@
         }
       }
     },
+    "UnresolvedDependenciesFilter": {
```

Comment:
The only property exposed here is count, however the example for apply for uses ResourceName.

Reply:
corrected. thanks